### DISCUSSION TITLE
How can I open another instance side by side with the original with its own settings?

### DISCUSSION THREAD

**Discussion by gerroon** [doublecmd/doublecmd#1343] [General]
Hi

I am running DC daily. I want to run another instance (side by side) with completely different settings, favorites, tabs etc How can I do it on Windows?

**Comment by Skif-off** [comment]
You can try to use [--config-dir=](https://doublecmd.github.io/doc/en/comandline.html).

**Reply by gerroon** [reply]
Thanks, I had to enable multiple instance running. It works with creating another shortcut for it.
